providers/k8s/resources/k8s.lr.go

```diff
@@ -6749,3 +6778,62 @@ func (c *mqlK8sUserinfo) GetUsername() *plugin.TValue[string] {
 func (c *mqlK8sUserinfo) GetUid() *plugin.TValue[string] {
 	return &c.Uid
 }
+
+// mqlK8sKubelet for the k8s.kubelet resource
+type mqlK8sKubelet struct {
+	MqlRuntime *plugin.Runtime
+	__id string
+	// optional: if you define mqlK8sKubeletInternal it will be used here
+	ConfigFile plugin.TValue[*mqlFile]
```

Comment:
I think this and the line below aren't correct because `file` and `process` are part of `os` provider